splunk_setup/savedsearches.conf

```diff
@@ -56,7 +56,7 @@ search = index="service_auth" failed password \
 | rex "Failed password for invalid user (?<fail_user>[^ ]+) from (?<ip>[^ ]+)" \
 | rex "message repeated (?<num_fails>[^ ]+) times: \[ Failed password for (?<fail_user>[^ ]+) from (?<ip2>[^ ]+)" \
 | eval ip=coalesce(ip, ip2) \
-| stats values(ip) as ip, sum(num_fails) as other_fails, count as fails by fail_user \
+| stats values(host) as hosts, values(ip) as ip, sum(num_fails) as other_fails, count as fails by fail_user \
 | eval other_fails=if(isnull(other_fails), 0, other_fails) \
 | eval total_fails=fails+other_fails \
 | where total_fails > 3 \
```

Comment:
Do we want to add `hosts` to the table as well?

Reply:
I dont see a reason not too